include/tvm/runtime/data_type.h

```diff
@@ -372,7 +372,7 @@ inline DLDataType String2DLDataType(std::string s) {
     t.lanes = 1;
     return t;
   } else if (s.substr(0, 6) == "bfloat") {
-    t.code = kTVMBFloat;
+    t.code = kDLBfloat;
```

Comment:
the type code should directly use DataType::kBfloat

Reply:
I agree with tq

Reply:
changed